src/Directory.Build.props

```diff
@@ -181,6 +181,7 @@
         <!-- Enable performance CA rules from 'Microsoft.CodeAnalysis.NetAnalyzers' as build warnings by default. Specific rules are disabled or downgraded in the repo's editorconfig. -->
         <AnalysisLevel>latest</AnalysisLevel>
         <AnalysisModePerformance>AllEnabledByDefault</AnalysisModePerformance>
+        <WarningsAsErrors>$(WarningsAsErrors);nullable</WarningsAsErrors>
```

Comment:
Why here and not as a unconditionally defined property? Is it because there's too much to do in samples or tests?

Reply:
@jeromelaban I was caring more about enable warnings as errors for the product code, though it seems there is only a single nullable warning in samples/test code. https://github.com/unoplatform/uno/pull/9963

Going to close this one since the other PR is going green.